index.js

```diff
@@ -0,0 +1,75 @@
+// EXERCISE 1: Selecting Elements
+// TODO: Select the element with the ID 'mainTitle' and log it to the console.
+let mainTitle = document.getElementById('mainTitle');
+console.log(mainTitle);
+// Exercise 2: Changing HTML Content
+// TODO: Select all paragraph elements and change the content of the second paragraph to "This paragraph has been changed."
+let paragraphs = document.getElementsByTagName('p');
+if (paragraphs.length >= 2) {
+    paragraphs[1].textContent = "This paragraph has been changed.";
+}
+
+// Exercise 3: Changing CSS Styles
+// TODO: Select the first list item and change its text color to red
+// TODO: Select the first list item and change its text color to red
+let firstListItem = document.querySelector('li');
+if (firstListItem) {
+    firstListItem.style.color = 'red';
+}
+
+// Exercise 4: Adding/Removing Classes
+// TODO: Add the "highlight" class to the first paragraph
+// TODO: Remove the "highlight" class from the first paragraph after 3 seconds
+let firstParagraph = document.querySelector('p');
+
+firstParagraph.classList.add('highlight');
+
+setTimeout(() => {
+    firstParagraph.classList.remove('highlight');
+}, 3000);
+
+// Exercise 5: Creating and Appending Elements
+// TODO: Create a new list item with the text "Item 4" and append it to the existing list
+let List = document.querySelector('ul');
+
+let newItem = document.createElement('li');
+
+newItem.textContent = 'Item 4';
+
+list.appendChild(newItem);
+
+// Exercise 6: Removing Elements
+// TODO: Remove the last list item from the unordered list
+let ListRemove = document.querySelector('ul');
```

Comment:
Avoid binding constant names to its implementation (`remove` in this case).

- [ ] Improve naming